test/net/sourceforge/kolmafia/preferences/PreferencesTest.java

```diff
@@ -440,6 +449,20 @@ void resetAscensionProperties() {
     assertNotEquals(val, Preferences.getInteger(name));
   }
 
+  void resetRolloverProperties() {
```

Comment:
This needs an @Test annotation, and could use a better name

Reply:
Renamed to testRolloverPropertiesAreReset and annotated